src/screens/ENSConfirmRegisterSheet.js

```diff
@@ -63,11 +63,7 @@ export default function ENSConfirmRegisterSheet() {
     yearsDuration: duration,
   });
   const prevStepGasLimit = usePrevious(stepGasLimit);
-
-  const { blurFields, values } = useENSProfileForm();
-  useFocusEffect(() => {
```

Comment:
@jxom this was causing some issues on the records update, I would open this sheet and the records were being erased, am I missing something?

Reply:
hmm, it shouldn't do that. might have been a merge issue upstream... i can have a look